### ISSUE TITLE
Pages with sidebar options lack skip-links

### ISSUE THREAD

**Issue by alexdryden** [UIUCLibrary/arclight#22]
Repository show with facets and Collection document show – Keyboard users must navigate through all main content before reaching the sidebar options. It would improve usability to place the sidebar first and add a “Skip to content” button [(Skip to info):](https://accessibleit.disability.illinois.edu/skipto-js-the-modern-way-to-implement-bypass-blocks/)

```
<body>
  <aside id="sidebar">...</aside>
  <main id="content">...</main>
</body>
```

**Comment by alexdryden** [comment]
@vicriveros I'm working on the a11y issues with ASpace you identified last month (https://github.com/UIUCLibrary/arcuit/pull/7#issuecomment-3168278098) and I just wanted to clarify my understanding of the desired state/behavior. 

Your suggestion is to move the sidebar content before the main content--is that so that keyboard users can then use shift-tab to tab up into the the sidebar content if they use the skip link to skip to the main content? 

If I'm not making 100% sense, I've added a brief screen recording that hopefully clarifies. 

https://github.com/user-attachments/assets/4bbe2ca4-6554-4b68-b391-11f775256334

**Comment by vdriveros** [comment]
Hi @alexdryden! Yes, that’s right. The idea is to place the sidebar first in the code structure since that’s how the user visually encounters it on the page. If the sidebar were on the right side, it would make sense to leave it after the main content in the code. By structuring it this way, if a user wants to skip past the sidebar and go straight to the main content, they can simply use the “Skip to content” link, which will take them directly to the `<main>`.

**Comment by alexdryden** [comment]
That makes sense! Thanks @vicriveros !
